README.md

```diff
@@ -161,6 +163,76 @@
 
 <hr>
 
+# Installation on Windows
+  **PLEASE PAY ATTENTION: RealSense ROS2 Wrapper is not meant to be supported on Windows by our team, since ROS2 and its packages are still not fully supported over Windows. We are putting these installation steps below in order to try and make it easier for users who already started working with ROS2 on Windows and want to take advantage of the capabilities of our RealSense cameras**
+
+<details>
+  <summary>
+    Step 1: Install the ROS2 distribution 
+  </summary>
+  
+- #### Windows 10/11
+  **Please choose only one option from the two options below (in order to prevent multiple versions installation and workspace conflicts)**
+  - Manual install from ROS2 formal documentation:
+    - [ROS2 Iron](https://docs.ros.org/en/iron/Installation/Windows-Install-Binary.html)
+	- [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Windows-Install-Binary.html)
+  - Microsoft IOT binary installation:
+    - https://ms-iot.github.io/ROSOnWindows/GettingStarted/SetupRos2.html
+    - Pay attention that the examples of install are for Foxy distro (which is not supported anymore by RealSense ROS2 Wrapper)
+	- Please replace the word "Foxy" with Humble or Iron, as you choose
+</details>
+  
+<details>
+  <summary>
+    Step 2: Download RealSense&trade; ROS2 Wrapper and RealSense&trade; SDK 2.0 source code from github:
+  </summary>
+  
+- Download the latest [Intel&reg; RealSense&trade; ROS2 Wrapper 4.54.1](https://github.com/IntelRealSense/realsense-ros/tree/4.54.1)

```

Comment:
Why should we point to a specific version if we instruct to take the latest?

Reply:
This is the latest released.. what you propose here ? to take the master from librealsense and ros2-development ? or what ?

Reply:
We can point to the ros releases page: https://github.com/IntelRealSense/realsense-ros/releases
And tell them to also get the matching LibRS supported release from the releases page: https://github.com/IntelRealSense/librealsense/releases
![image](https://github.com/IntelRealSense/realsense-ros/assets/64067618/82eca3a0-d56a-4cdd-89cb-e1c4ac29c32c)

Reply:
done, please review last commit